code/modules/clothing/outfits/ert.dm

```diff
@@ -333,3 +333,29 @@
 	suit_store = /obj/item/gun/ballistic/rifle/boltaction
 	r_hand = /obj/item/megaphone
 	head = /obj/item/clothing/head/intern
+
+/datum/outfit/doomguy
+	name = "The Juggernaut"
+
+	uniform = /obj/item/clothing/under/rank/centcom/officer
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/doomguy
+	shoes = /obj/item/clothing/shoes/jackboots/fast
+	gloves = /obj/item/clothing/gloves/krav_maga/combatglovesplus
```

Comment:
might wanna mention that Doomguy gets krav maga in the PR description

Reply:
Done